tests/test_wave_01.py

```diff
@@ -158,13 +158,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
 
-@pytest.mark.skip()
+    # Our assertions (asserts that the correct movie was added to watched)
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
+    assert updated_data["watched"][0]["genre"] == GENRE_1
+    assert updated_data["watched"][0]["rating"] == pytest.approx(RATING_1)
+
```

Comment:
Does the rating need to be approximately or exact? We aren't calculating anything so we should expect it to be exactly what `RATING_1` is. 
```suggestion
    # Our assertions (asserts that the correct movie was added to watched)
    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
    assert updated_data["watched"][0]["genre"] == GENRE_1
    assert updated_data["watched"][0]["rating"] == RATING_1

```